docs/eventing/migration/sinkbinding.md

```diff
@@ -15,7 +15,7 @@ In YAML, these two options are equivalent:
 1. `ContainerSource` that emits events to a `Knative Service`:
 
     ```yaml
-    apiVersion: sources.knative.dev/v1alpha2
+    apiVersion: sources.knative.dev/v1beta1
```

Comment:
@nachocano Shouldn't this be done for sinkbinding as well (on line 40 in this file)